From 5e3a0d2a8ad7da58ae112b10028986eab7e4ae89 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Tue, 7 Feb 2017 22:48:45 +0900
Subject: [PATCH 1811/2066] Revert "clk: renesas: r8a7795: Add MFIS clock"

This reverts commit 95f0b879197f ("clk: renesas: r8a7795: Add MFIS
clock").

Although it is necessary to start the MFISLock driver before each
driver's start when performing exclusive control of register access by
MFISLock driver, since CPG driver I/F is used for the MFIS clock control
of MFIS driver, the MFIS driver is started after CPG driver start.

Therefore, the problem whose exclusive control is impossible to access
register until the MFISLock driver starts occurs.

By this revert, MFIS clock control becomes no good by deleting the
definition of the MFIS clock from the CPG driver.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/clk/renesas/r8a7795-cpg-mssr.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/clk/renesas/r8a7795-cpg-mssr.c b/drivers/clk/renesas/r8a7795-cpg-mssr.c
index 41a1a79..69cb1ad 100644
--- a/drivers/clk/renesas/r8a7795-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7795-cpg-mssr.c
@@ -136,7 +136,6 @@ static struct mssr_mod_clk r8a7795_mod_clks[] __initdata = {
 	DEF_MOD("msiof2",		 209,	R8A7795_CLK_MSO),
 	DEF_MOD("msiof1",		 210,	R8A7795_CLK_MSO),
 	DEF_MOD("msiof0",		 211,	R8A7795_CLK_MSO),
-	DEF_MOD("mfis",			 213,	R8A7795_CLK_S3D2),
 	DEF_MOD("sys-dmac2",		 217,	R8A7795_CLK_S0D3),
 	DEF_MOD("sys-dmac1",		 218,	R8A7795_CLK_S0D3),
 	DEF_MOD("sys-dmac0",		 219,	R8A7795_CLK_S0D3),
-- 
1.7.5.4

